common/src/main/java/com/wynntils/utils/type/NamedValue.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.utils.type;
+
+public record NamedValue(String name, int value) {
```

Comment:
Should we make the value a double?

Reply:
Let's start with having it an int, shall we? If it turns out to be insufficient, then we can change it.

Reply:
Okay